packages/perseus/src/widgets/interactive-graphs/graphs/angle.tsx

```diff
@@ -89,16 +85,102 @@ export function AngleGraph(props: AngleGraphProps) {
         <>
             {svgLines}
             <Angle {...angleParams} />
-            {coords.map((point, i) => (
-                <StyledMovablePoint
-                    key={"point-" + i}
-                    snapTo={"angles"}
-                    point={point}
-                    onMove={(destination: vec.Vector2) =>
-                        dispatch(actions.angle.movePoint(i, destination))
-                    }
-                />
-            ))}
+            {/* vertex */}
+            <MovablePoint
```

Comment:
Also appreciate these callouts to help explain which points they are, now that we're not mapping them.